src/react-guides/Guides.tsx

```diff
@@ -523,4 +519,14 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
             selectedGuides:[],
         });
     }
+
+    private calcHorizontalTransform(offsetX: number, nextPos: number) {
+        const displayPos = [offsetX, nextPos];
+        return `translate(${displayPos.map((v, i) => !i ? '-6px' : `${v - 10}px` ).join(', ')}) rotate(-90deg)`;
+    }
+
+    private calcVerticalTransform(nextPos: number, offsetY: number) {
```

Comment:
I do not understand why you change the order of arguments.
It seems like your functions are pretty similar, but in this place offsetY is second while offsetX is first in the previous function